File: python/svix/models/stream_in.py
# this file is @generated
import typing as t

from .common import BaseModel


class StreamIn(BaseModel):
    metadata: t.Optional[t.Dict[str, str]] = None

    name: str
    """The stream's name."""

    uid: t.Optional[str] = None
    """An optional unique identifier for the stream."""

File: kotlin/lib/src/main/kotlin/models/StreamIn.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class StreamIn(
    val metadata: Map<String, String>? = null,
    /** The stream's name. */
    val name: String,
    /** An optional unique identifier for the stream. */
    val uid: String? = null,
)
